### PR TITLE
First version of particles landing page

### PR DESCRIPTION
**Pull request by StephenBlank**

### CODE REVIEW

**Comment by GeorgeDover** [comment]
Looks good

File: README.md [hunk 1]
```diff
@@ -1,2 +1,70 @@
-# FormulaicLLC.github.io
-FormulaicLLC Website
```
Comment (GeorgeDover) [inline README.md:2]: Should we revert back to this README content?

**Review by GeorgeDover — CHANGES_REQUESTED** [review]
README.md needs review

File: README.md (hunk 1 above)
Comment (StephenBlank) [reply README.md:2]: yes, I agree

**Review by GeorgeDover — APPROVED** [review]
looks good

**Pull request merged**